packages/editor/src/functions/assetFunctions.ts

```diff
@@ -1,5 +1,27 @@
 import { client } from '@xrengine/client-core/src/feathers'
 import { uploadToFeathersService } from '@xrengine/client-core/src/util/upload'
+import { EntityTreeNode } from '@xrengine/engine/src/ecs/classes/EntityTree'
+import { getComponent } from '@xrengine/engine/src/ecs/functions/ComponentFunctions'
+import { AssetComponent } from '@xrengine/engine/src/scene/components/AssetComponent'
+import { Object3DComponent, Object3DWithEntity } from '@xrengine/engine/src/scene/components/Object3DComponent'
+import { sceneToGLTF } from '@xrengine/engine/src/scene/functions/GLTFConversion'
+
+import { accessEditorState } from '../services/EditorServices'
+
+export const exportAsset = async (node: EntityTreeNode) => {
+  const ass = getComponent(node.entity, AssetComponent)
```

Comment:
lets use more descriptive full variable names here. `assetComponent` instead of `ass`